### ISSUE TITLE
Installation errors on U22 with 8.3.1

### ISSUE THREAD

**Issue by rhavern** [FoldingAtHome/fah-client-bastet#196]
This was observed in previous iterations:
```
rick@Server2:~/Downloads$ sudo dpkg -i fah-client_8.3.1_amd64.deb
Selecting previously unselected package fah-client.
(Reading database ... 207227 files and directories currently installed.)
Preparing to unpack fah-client_8.3.1_amd64.deb ...
Failed to kill unit FAHClient.service: Unit FAHClient.service not loaded.
Unpacking fah-client (8.3.1) ...
Setting up fah-client (8.3.1) ...

The Folding@home client is now installed

File locations:

  Logs: /var/log/fah-client
  Data: /var/lib/fah-client

Service commands:

  systemctl status --no-pager -l fah-client
  sudo systemctl start fah-client
  sudo systemctl stop fah-client
  sudo systemctl restart fah-client

Access the web interface by going to:

  https://alpha.foldingathome.org/

Processing triggers for mailcap (3.70+nmu1ubuntu1) ...
Processing triggers for gnome-menus (3.36.0-1ubuntu3) ...
Processing triggers for desktop-file-utils (0.26-1ubuntu3) ...
```

After multiple uninstall, reboot, install cycles:
```
rick@Server2:~/Downloads$ sudo dpkg -D2 -i fah-client_8.3.1_amd64.deb
[sudo] password for rick:
(Reading database ... 207245 files and directories currently installed.)
Preparing to unpack fah-client_8.3.1_amd64.deb ...
D000002: fork/exec /var/lib/dpkg/info/fah-client.prerm ( upgrade 8.3.1 )
D000002: fork/exec /var/lib/dpkg/tmp.ci/preinst ( upgrade 8.3.1 8.3.1 )
Unpacking fah-client (8.3.1) over (8.3.1) ...
D000002: fork/exec /var/lib/dpkg/info/fah-client.postrm ( upgrade 8.3.1 )
D000002: process_archive info installed /var/lib/dpkg/tmp.ci/prerm as /var/lib/dpkg/info/fah-client.prerm
D000002: process_archive info installed /var/lib/dpkg/tmp.ci/postinst as /var/lib/dpkg/info/fah-client.postinst
D000002: process_archive info installed /var/lib/dpkg/tmp.ci/postrm as /var/lib/dpkg/info/fah-client.postrm
D000002: process_archive info installed /var/lib/dpkg/tmp.ci/copyright as /var/lib/dpkg/info/fah-client.copyright
D000002: process_archive info installed /var/lib/dpkg/tmp.ci/preinst as /var/lib/dpkg/info/fah-client.preinst
D000002: process_archive info unlinked /var/lib/dpkg/info/fah-client.md5sums
D000002: process_archive tmp.ci script/file '.' contains dot
D000002: process_archive tmp.ci script/file '..' contains dot
D000002: process_archive tmp.ci script/file '/var/lib/dpkg/tmp.ci/control' is control
Setting up fah-client (8.3.1) ...
D000002: fork/exec /var/lib/dpkg/info/fah-client.postinst ( configure 8.3.1 )

The Folding@home client is now installed

File locations:

  Logs: /var/log/fah-client
  Data: /var/lib/fah-client

Service commands:

  systemctl status --no-pager -l fah-client
  sudo systemctl start fah-client
  sudo systemctl stop fah-client
  sudo systemctl restart fah-client

Access the web interface by going to:

  https://alpha.foldingathome.org/

Processing triggers for mailcap (3.70+nmu1ubuntu1) ...
D000002: fork/exec /var/lib/dpkg/info/mailcap.postinst ( triggered /usr/share/applications )
Processing triggers for gnome-menus (3.36.0-1ubuntu3) ...
D000002: vmaintscript_installed nonexistent postinst
Processing triggers for desktop-file-utils (0.26-1ubuntu3) ...
D000002: fork/exec /var/lib/dpkg/info/desktop-file-utils.postinst ( triggered /usr/share/applications )
rick@Server2:~/Downloads$ systemctl status --no-pager -l fah-client
â fah-client.service - Folding@home Client
     Loaded: loaded (/lib/systemd/system/fah-client.service; disabled; vendor preset: enabled)
     Active: inactive (dead)
rick@Server2:~/Downloads$ sudo systemctl start fah-client
rick@Server2:~/Downloads$ systemctl status --no-pager -l fah-client
â fah-client.service - Folding@home Client
     Loaded: loaded (/lib/systemd/system/fah-client.service; disabled; vendor preset: enabled)
     Active: active (running) since Thu 2024-01-11 12:27:22 GMT; 3s ago
   Main PID: 12656 (fah-client)
      Tasks: 2 (limit: 7009)
     Memory: 4.3M
        CPU: 22ms
     CGroup: /system.slice/fah-client.service
             ââ12656 /usr/bin/fah-client --config=/etc/fah-client/config.xml --log=/var/log/fah-client/log.txt --log-rotate-dir=/var/log/fah-client/

Jan 11 12:27:22 Server2 systemd[1]: Started Folding@home Client.
```

**Comment by marcosfrm** [comment]
`systemctl` used to be silent when killing inactive init scripts.

Version 239 from RHEL 8:

```
$ systemctl status FAHClient.service 
* FAHClient.service - LSB: Folding@home Client
   Loaded: loaded (/etc/rc.d/init.d/FAHClient; generated)
   Active: inactive (dead)
     Docs: man:systemd-sysv-generator(8)
$ sudo systemctl kill FAHClient.service 
$
```

Version 252 from Debian 12:

```
$ systemctl status FAHClient.service 
* FAHClient.service - LSB: Folding@home Client
     Loaded: loaded (/etc/init.d/FAHClient; generated)
     Active: inactive (dead)
       Docs: man:systemd-sysv-generator(8)
$ sudo systemctl kill FAHClient.service 
Failed to kill unit FAHClient.service: Unit FAHClient.service not loaded.
```
(probably a bug in `systemctl`)

I could not reproduce the `try-restart` error you posted on Slack.

**Comment by jcoffland** [comment]
This should be fixed in 8.3.2.
